draft-cpbs-pseudorandom-ctls.md

```diff
@@ -190,15 +178,23 @@ In general, a malicious peer can still produce desired ciphertext with probabili
 
 Representing plaintext alerts (i.e. CTLSPlaintext messages with `content_type = alert(21)`) requires additional steps, because Alert fragments have little entropy.
 
-A standard TLS Alert fragment is always 2 bytes long.  In Pseudorandom cTLS, senders MUST prepend at least 16 random bytes to any plaintext Alert fragment.  Enciphering and deciphering then proceed identically to other CTLSPlaintext messages.  The recipient MUST remove these bytes before passing the CTLSPlaintext to the cTLS implementation.
-
-Servers SHOULD expand any Alert message following the ClientHello to the same size as their usual ServerHello, and SHOULD send additional random TCP segments or datagrams to match the sizes of subsequent components of their ordinary success response.  Otherwise, an adversary could use probing to learn the allowed lengths of ClientHellos and the fraction of ciphertexts that decipher to valid ClientHellos.
```

Comment:
@cjpatton I think most "controversial" change here is to remove this recommendation, and instead tell servers to authenticate the ClientHello before responding.  I think this is probably the wisest solution due to the [Parrot Is Dead](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.686.1317&rep=rep1&type=pdf) problem.

Reply:
Yup, I agree.